kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -509,13 +505,13 @@
     methods: {
       handleReplaceSelection() {
         const section_id = get(this.activeSection).section_id;
-        this.$router.replace({ path: 'new/' + section_id + '/replace-questions' });
+        this.$router.push({ path: 'new/' + section_id + '/replace-questions' });
```

Comment:
I hadn't noticed this before - but is there a reason to use a path here and below, rather than `name` and `param`? It feels a little less brittle to do the latter, so if we need to tweak URLs for whatever reason, this remains unchanged.

Reply:
Thanks! I noticed that in passing yesterday while focused on another part of the code and failed to come back to fix it up. Will fix shortly